softlayer/vm/interface.go

```diff
@@ -8,14 +8,27 @@ import (
 )
 
 type VMCloudProperties struct {
-	Domain                   string `json:"domain,omitempty"`
-	StartCpus                int    `json:"startCpus,omitempty"`
-	MaxMemory                int    `json:"maxMemory,omitempty"`
-	Datacenter               sldatatypes.Datacenter
-	BlockDeviceTemplateGroup sldatatypes.BlockDeviceTemplateGroup
-	SshKeys                  []sldatatypes.SshKey `json:"sshKeys"`
-	RootDiskSize             int                  `json:"rootDiskSize,omitempty"`
-	EphemeralDiskSize        int                  `json:"ephemeralDiskSize,omitempty"`
+	Domain                   string 													`json:"domain,omitempty"`
+	StartCpus                int    													`json:"startCpus,omitempty"`
+	MaxMemory                int    													`json:"maxMemory,omitempty"`
+	Datacenter               sldatatypes.Datacenter										`json:"datacenter"`
+	BlockDeviceTemplateGroup sldatatypes.BlockDeviceTemplateGroup						`json:"blockDeviceTemplateGroup,omitempty"`
+	SshKeys                  []sldatatypes.SshKey 										`json:"sshKeys"`
+	RootDiskSize             int                  										`json:"rootDiskSize,omitempty"`
+	EphemeralDiskSize        int                  										`json:"ephemeralDiskSize,omitempty"`
+
+	//added by wenquan@cn.ibm.com
```

Comment:
No need to include this type of comment. Your commit will indicate who you are and what you added. Thanks.